src/components/organisms/Header/Header.tsx

```diff
@@ -133,10 +144,16 @@ const Header = ({ showNavigation }: { showNavigation: boolean }) => {
                         )}
                     </div>
                     <div className='col-span-2 flex flex-row items-center justify-end gap-2 laptop:col-span-1 laptop:gap-2.5'>
-                        <PointsTag
-                            isConnected={verifiedData && address && userData}
-                            points={userPoints}
-                        />
+                        {isShowStablecoinLink && <StablecoinExternalLink />}
+                        {/* condition: only show Points tag in mobile if show stablecoin link is true  */}
```

Comment:
think this comment might be incorrect: 
condition: only show Points tag in mobile if show stablecoin link is "false"

Reply:
It is expected. There are 2 cases

1) If stablecoin link is shown, hide points tag in mobile
2) If stablecoin isn't shown, show points tag in mobile

And the points tag is shown in laptop all the time

Reply:
Yep. But your comment is incorrect.

Reply:
Removing comment. Implementation is ok, yes?